src/main/java/Duke/TaskCreator.java

```diff
@@ -0,0 +1,24 @@
+package Duke;
+
+import java.util.Arrays;
+
+public class TaskCreator {
+    public static Task createNewTask(String[] commandByWord)
+            throws IllegalArgumentException, ArrayIndexOutOfBoundsException {
+        String type = commandByWord[0];
+        if (type.equals("todo")) {
+            return ToDo.createToDo(commandByWord);
+        } else if (type.equals("deadline")) {
+            return Deadline.createDeadline(commandByWord);
+        } else {
```

Comment:
You should not default your command behaviours like this! Create a `if(condition)` for each command :)